docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Creating Accounts
+sidebar_position: 2
+---
+
+Accounts in **ENGAGE** are separate individuals (donors or supporters) and organizations that donate charity. This article will walk you through creating a new account via ENGAGE.
+
+## Option 1:1 Create an Account from the Dashboard using Create
+
+- On the [ENGAGE dashboard](https://beta.n3o.cloud/eu1/engage/dashboard), click the blue button **Create**. A bar containing different icons will be shown.
```

Comment:
I would avoid referencing things like button colours as those are highly likely to change and we won't keep the docs in sync.

If you can also change the 1:1 and 1:2 in the headings. This reads odd (at least in British English). I think you mean it as "1 of 2" and "2 of 2" but for example 1:1 is ready as one-to-one as in a 1:1 relationship. I would also avoid repeating the page title so these should just be "Option 1: Create from the dashboard" and "Option 2: Create from quick navigation".

We also need to agree capitalisation. I don't think we should capitalise things like Dashboard and Quick Navigation.